packages/integrations/useSortable/index.md

```diff
@@ -0,0 +1,99 @@
+---
+category: '@Integrations'
+---
+
+# useSortable
+
+Wrapper for [`sortable`](https://github.com/SortableJS/Sortable).
+
+## Install 
+
+```bash
+npm i sortablejs
+```
+
+## Usage
+
```

Comment:
We can add a first explanation, for example:

Use a `template ref` on the root element to make the list sortable.